common/utils/src/main/resources/error/error-classes.json

```diff
@@ -500,6 +500,24 @@
     ],
     "sqlState" : "22004"
   },
+  "COMPLEX_EXPRESSION_UNSUPPORTED_INPUT" : {
+    "message" : [
+      "Cannot process input data types for the expression: <expression>."
+    ],
+    "subClass" : {
+      "MISMATCHED_TYPES" : {
+        "message" : [
+          "All input types must be the same except nullable, containsNull, valueContainsNull flags, but found the input types <inputTypes>."
+        ]
+      },
+      "NO_INPUTS" : {
+        "message" : [
+          "The collection of input data types must not be empty."
+        ]
+      }
+    },
+    "sqlState" : "42K09"
```

Comment:
@srielau WDYT of this state? Shouldn't we introduce a separate error class for require.

Reply:
I'm not sure about the `sqlState`, but I don't think we should introduce a separate error class for `require` because there can be different `require` messages/errors depending on what was checked. This specific error class isn't for all `require`, it is just for one.